aiida_quantumespresso/parsers/pp.py

```diff
@@ -0,0 +1,348 @@
+# -*- coding: utf-8 -*-
+""" Parser for pp.x calculations """
+from __future__ import absolute_import
+
+import traceback
+
+import numpy as np
+
+from aiida import orm
+from aiida.common import exceptions
+
+from aiida_quantumespresso.calculations.pp import PpCalculation
+from aiida_quantumespresso.utils.mapping import get_logging_container
+from .base import Parser
+
+from six.moves import range
+from six.moves import zip
+
+class PpParser(Parser):
+    """
+    Parser for pp.x calculations
+    """
+
+    # Lookup for units
+    units_dict = {
+        0: 'e/bohr^3', # Electrons, electronic charge density
+        1: 'Ry', # Total potential
+        2: 'Ry', # Ionic potential
+        3: 'states/eV.bohr^3', # Density of states over an energy range
+        4: 'states/eV.bohr^3', # Local density of electronic entropy
```

Comment:
Well, what is entropy in Rydberg units? Should be energy per temperature, right? So.. maybe Ry/K.bohr^3?

Reply:
> To be honest, I also haven't used most of the plotting kinds - but I think we should do some more checking on these units (maybe ask someone who knows).

Yes, looking again, I'm seeing that it's a bit of a mess from rushing.

Reply:
> Well, what is entropy in Rydberg units? Should be energy per temperature, right? So.. maybe Ry/K.bohr^3?

I'll make it this for now